src/app/common/interceptors/getUser.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  'use strict';
+  angular
+  .module('healthGuide')
+    .factory('getUserInterceptor', getUserInterceptor);
```

Comment:
service that return inctance of itself. Isn't it factory? 
Plus Angular docs [provides example with factory](https://docs.angularjs.org/api/ng/service/$http)
`$provide.factory('myHttpInterceptor',...`